number-of-operations-to-make-network-connected.cpp

```diff
@@ -0,0 +1,49 @@
+// Author: Denis Davidoglu
+// Question Link: https://leetcode.com/problems/number-of-operations-to-make-network-connected/
+

```

Comment:
You can add time and memory complexities if you want, other than that it is perfect.